ompi/mca/coll/han/coll_han_bcast.c

```diff
@@ -71,7 +77,7 @@ mca_coll_han_bcast_intra(void *buff,
 
     /* Create the subcommunicators */
     err = mca_coll_han_comm_create(comm, han_module);
-    if( OMPI_SUCCESS != err ) {  /* Let's hope the error is consistently returned across the entire communicator */
+    if( OMPI_SUCCESS != err ) {
```

Comment:
The return code is the same for all ranks since the only current error case is conditionned by a MPI_Allreduce on the communicator. 
https://github.com/open-mpi/ompi/blob/v4.1.x/ompi/mca/coll/han/coll_han_subcomms.c#L94
or 
https://github.com/open-mpi/ompi/blob/v4.1.x/ompi/mca/coll/han/coll_han_subcomms.c#L231

What may however be concerning is "Does all the ranks fallback on the same component?" (in case some of the environment is not uniform on all the nodes). But this is very unlikely and out of the scope of this PR.